vcpkg.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "webkit",
+  "dependencies": [
+    "icu"
+  ],
+  "features": {
+    "avif": {
+      "description": "Build with AVIF image support",
+      "dependencies": [
+        {
+          "name": "libavif",
+          "features": [
+            "aom"
+          ]
+        }
+      ]
+    },
```

Comment:
@fujii @tats-u with `vcpkg` you can add features for the build. In this case it'll use the upstream `vcpkg` registry to build with avif support.

The `build-webkit` script would need to have knowledge of this going forward.